pkg/apis/core/v1alpha2/types.go

```diff
@@ -155,3 +155,27 @@ type ClusterGroupList struct {
 
 	Items []ClusterGroup `json:"items,omitempty"`
 }
+
+// AppliedTo selects the entities to which a policy is applied.
+type AppliedTo struct {
+	// Selectors is the set of EntitySelectors that select entities.
+	// +optional
+	Selectors []EntitySelector `json:"selectors,omitempty"`
```

Comment:
@Dyanngg, @tnqn, @antoninbas  : do you see any reason to make it a slice? Today Antrea NetworkPolicy uses a slice of NetworkPolicyPeer so can be multiple Selectors too.

Reply:
I meant I am thinking about:
```
type AppliedTo struct {
        PodSelector *metav1.LabelSelector `json:"podSelector,omitempty"`
        NamespaceSelector *metav1.LabelSelector `json:"namespaceSelector,omitempty"`
        Selectors []EntitySelector `json:"selectors,omitempty"`
}
```

Reply:
Changed to a single selector (pair) per inputs from Abhishek and Yang.